README.md

```diff
@@ -1,745 +1,148 @@
-be strings or regular expressions.
+and you'll be all set to start programming.
 
-Can also be set via the environment variable `NEW_RELIC_IGNORING_RULES`, with
-multiple rules passed in as a list of comma-delimited patterns:
-`NEW_RELIC_IGNORING_RULES='^/socket\.io/\*/xhr-polling,ignore_me'` Note that
-currently there is no way to escape commas in patterns.
+To run the test suite
 
-### API for adding naming and ignoring rules
+1. [Install Docker](https://www.docker.com/products/docker-desktop)
+2. Start the docker services: `$ npm run services`
+3. Run all the tests via `$ npm run test`
 
-#### newrelic.addNamingRule(pattern, name)
+Available test suites include
 
-Programmatic version of `rules.name` above. Naming rules can not be removed
-until the Node process is restarted. They can also be added via the module's
-configuration. Both parameters are mandatory.
+    $ npm run unit
+    $ npm run integration
+    $ npm run versioned
+    $ npm run lint
+    $ npm run smoke
 
-#### newrelic.addIgnoringRule(pattern)
+## Further Reading
 
-Programmatic version of `rules.ignore` above. Ignoring rules can not be removed
-until the Node process is restarted. They can also be added via the module's
-configuration. The pattern is mandatory.
+Here's some resources for learning more about the Agent
 
-### Other API calls
+- [New Relic's official NodeJS Agent Documentation](https://docs.newrelic.com/docs/agents/nodejs-agent)
 
-#### newrelic.addCustomAttribute(name, value)
+- [Developer Docs](http://newrelic.github.io/node-newrelic/docs/)
 
-Set a custom attribute value to be attached to a transaction trace and/or error
-in the New Relic UI. This must be called within the context of a transaction,
-so it has a place to set the custom attributes.
+- [Configuring the Agent (via `newrelic.js` or environment variables)](https://docs.newrelic.com/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration)
 
-#### newrelic.addCustomAttributes(params)
+- [Use the Node Agent to add the Browser and SPA Monitoring](https://docs.newrelic.com/docs/agents/nodejs-agent/supported-features/new-relic-browser-nodejs-agent)
 
-Set multiple custom attribute values to be attached to a transaction trace and/or
-error in the New Relic UI. This must be called within the context of a transaction,
-so it has a place to set the custom attribute.
+- [API Transaction Naming](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#request-names) and [Rules Based Transaction Naming](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#ignoring)
 
-Example of setting multiple custom attribute at once:
+- [Custom Instrumentation/Transactions](https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api#creating-transactions)
 
-```js
-newrelic.addCustomAttributes({test: 'value', test2: 'value2'});
-```
+- [The Changelog](/node-newrelic/blob/main/NEWS.md)
```

Comment:
This link does not work in `rich diff` view.  I believe it should just be `News.md`